model/src/main/kotlin/utils/FindingsMatcher.kt

```diff
@@ -103,12 +103,12 @@ class FindingsMatcher(
         // If there are multiple license findings in a single file, search for the closest copyright statements
         // for each of these, if any.
         val copyrightsForLicenses = mutableMapOf<String, MutableSet<CopyrightFindings>>()
-        licenses.forEach {
+        licenses.forEach { licenseFinding ->
```

Comment:
Following the IDE hint here to create a destructing declaration like `(license, location) ->` makes the code a tad more readable.

Reply:
Done. Even though for this IDE hint I'm not 100% convinced whether the destructing should be always used as it is a bit more fragile and probably has less IDE refactoring support.

Reply:
I totally agree that we should consider following this IDE hint on a case-by-case basis, as often following it makes the code actually *less* readable IMO, but in this case I applied the hint to confirm it improves readability.

Reply:
Done.

Reply:
Now the destructing declaration from the previous commit is lost, but it is as applicable here as before, and should be kept IMO.

Reply:
Done.